.github/workflows/marketplace-publish.yml

```diff
@@ -22,7 +23,7 @@ jobs:
 
       - uses: actions/setup-node@v4
         with:
-          node-version: 18
+          node-version: ${{ env.NODE_VERSION }}
```

Comment:
@mrubens - LMK if you think this is potentially problematic. I'll test a production build locally as well.